translations/ai_heroes.tex/pl.po

```diff
@@ -58,6 +58,16 @@ msgid ""
 "\\end{itemize}\n"
 "\n"
 msgstr ""
+"\\subsection*{Tura Bohatera AI}\n"
+"\\hypertarget{AIrules}{Bohaterowie AI} występują w Kampaniach.\n"
+"Bohater AI w Trybie Solo zaczyna grę w swoim Mieście, posiada 3 Punkty Ruchu (PR), przeznaczone do następujących działań według schematu:\n"
+"\\begin{itemize}\n"
+"  \\item Jeśli Bohater Gracza znajduje się na tym samym Kafelku, co Bohater AI, wykorzystaj PR, przemieść się w jego kierunku i rozpocznij Walkę.\n"
```

Comment:
```suggestion
"  \\item Jeśli Bohater gracza znajduje się na tym samym Kafelku, co Bohater AI, wykorzystaj PR, aby przemieścić się w jego kierunku w celu rozpocznęcia Walki.\n"
```
To zdanie jest nieprecyzyjne. Bohaterowi AI może nie wystarczyć punktów ruchu, aby rozpocząć walkę.